src/Containers/Listing/ListingDetail.js

```diff
@@ -137,9 +135,19 @@ export default function ListingDetail(props) {
                     data[item.id] = parseInt(item.value);
                     return;
                   }
-    
                   data[item.id] = item.value;
                 });
+
+                const addr = data['listingAddress'];
+                APIs.getValidatedAddress(addr).then(resp => {
```

Comment:
check if address is validated?
if so, perform action.
if not, do not close the modal, and set snack bar sate with `warning` severity

The key is to prevent user from submitting address that's not validated

Reply:
可是有个问题，就是address verification只有form submit了才能verify。。。

Reply:
阻止不了invalid address被write in